components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -23,15 +45,33 @@ use tikv_util::{box_err, time::monotonic_raw_now};
 
 use crate::{
     batch::StoreContext,
-    fsm::{PeerFsm, PeerFsmDelegate},
-    raft::Peer,
-    router::CmdResChannel,
+    fsm::{ApplyFsm, ApplyResReporter, PeerFsmDelegate},
+    raft::{Apply, Peer},
+    router::{ApplyRes, ApplyTask, CmdResChannel, PeerMsg},
 };
 
 mod write;
 
 pub use write::{SimpleWriteDecoder, SimpleWriteEncoder};
 
+use self::write::SimpleWrite;
+
+#[derive(Debug)]
+pub struct CommittedEntries {
+    /// Entries need to be applied. Note some entries may not be included for
```

Comment:
I don't understand again... Are these entries delayed or? Why is it necessary to comment this detail (i.e. where could rely on the "wholeness" of CommittedEntries?)

Reply:
Yes, those entries are delayed. There is an applying entries window.

> where could rely on the "wholeness" of CommittedEntries

It's wrong to assume the last index of `entry_and_proposals` is the latest commit index. raftstore-v2 doesn't need latest committed index during apply anymore. But better keep the comment in case we want committed index again.